packages/typescript-estree/tests/lib/persistentParse.ts

```diff
@@ -7,6 +7,7 @@ const CONTENTS = {
   foo: 'console.log("foo")',
   bar: 'console.log("bar")',
   'baz/bar': 'console.log("baz bar")',
+  'ui/components/file': 'console.log("ui/components")',
```

Comment:
hmmm.
I think that you could probably achieve this via fixing the "allows parsing of deeply nested new files" test in `baseTests`.

I.e. change `baz/bar` to `baz/bar/foo`, then add a new test:
```ts
describe('glob config one level deep', () => {
    const tsConfigExcludeBar = {
      include: ['./**/*'],
      exclude: ['./src/bar.ts'],
    };
    const tsConfigIncludeAll = {
      include: ['./**/*'],
    };

    baseTests(tsConfigExcludeBar, tsConfigIncludeAll);
  });
```

Reply:
This is not correct, this issue happens only if extends is used